concepticondata/conceptlists.tsv

```diff
@@ -291,13 +291,13 @@ Schroeder-2012-824	Schroeder, Astrid and Gemballa, Teresa and Ruppin, Steffie an
 Desrochers-2010-330	Desrochers, Alain and Liceras, Juana M. and Fernandez-Fuertes, Raquel and Thompson, Glenn L.	2010		330	ranked	Spanish	Spanish	https://link.springer.com/article/10.3758/BRM.42.1.109#SecESM1	Desrochers2010		Subjective frequency ratings from 102 Spanish native speakers of 330 words (compounds, nouns and verbs). Objective frequency norms and mean subjective familiarity ratings were drawn from the Spanish lexical database LEXESP [Sebastian-Galles et al., 2000](:bib:Sebastian2000).		
-Monnier-2014-1031	Monnier, Catherine and Syssau, Arielle	2014		1031	ranked	French	French	https://link.springer.com/article/10.3758/s13428-013-0431-1#SupplementaryMaterial	Monnier2014		The stimulus set contained 1,031 French words (969 nouns and 62 adjectives). The words in FAN cover a wide range of categories (objects, animals, food, and concepts). Ratings were made using the Self-Assessment Manikin [(Lang, 1980)](:bib:Lang1980).  Although the materials were words, the column PICTURE_SOURCE lists picture sources, when available (“AF” for [Alario & Ferrand, 1999](:bib:Alario1999); “B” for [Bonin, Peereman, et al. 2003](:bib:Bonin2003)). Six additional objective psycholinguistic indices were taken from Lexique 3.80 [(New et al., 2004)](:bib:New2004). The ratings for the subjective psycholinguistic index imageability were taken from [Bonin, Méot, Ferrand, and Roux (2011)](:bib:Bonin2011), [Desrochers and Bergeron (2000)](:bib:Desrochers2000), and [Gonthier, Desrochers, Thompson, and Landry (2009)](:bib:Gonthier2009). 		
+Monnier-2014-1031	Monnier, Catherine and Syssau, Arielle	2014		1031	ranked	French	French	https://link.springer.com/article/10.3758/s13428-013-0431-1#SupplementaryMaterial	Monnier2014		The stimulus set contained 1,031 French words (969 nouns and 62 adjectives). The words in FAN cover a wide range of categories (objects, animals, food, and concepts). Ratings were made using the Self-Assessment Manikin [(Lang, 1980)](:bib:Lang1980).  Although the materials were words, the column PICTURE_SOURCE lists picture sources, when available (“AF” for [Alario & Ferrand, 1999](:bib:Alario1999); “B” for [Bonin, Peereman, et al. 2003](:bib:Bonin2003)). Six additional objective psycholinguistic indices were taken from Lexique 3.80 [(New et al., 2004)](:bib:New2004). The ratings for the subjective psycholinguistic index imageability were taken from [Bonin, Méot, Ferrand, and Roux (2011)](:bib:Bonin2011), [Desrochers and Bergeron (2000)](:bib:Desrochers2000), and [Gonthier, Desrochers, Thompson, and Landry (2009)](:bib:Gonthier2009).		
```

Comment:
@MacyL, please could you check the difference between these lines? Is it additional tabstops, or missing ones? It is important to check these diffs.

Reply:
I guess it's Mac numbers added some quotation marks during export, I will check the file and fix it.